src/utils/vcs.rs

```diff
@@ -587,10 +587,22 @@ use {
 
 #[test]
 fn test_find_matching_rev_with_lightweight_tag() {
+    let dir = tempdir().expect("Failed to generate temp dir.");
+    git_initialize_repo(dir.path());
+
+    git_create_commit(
+        dir.path(),
+        "foo.js",
+        b"console.log(\"Hello, world!\");",
```

Comment:
@kamilogorek why is this a bytestring?

Reply:
oh I saw further down that it's just any content.

Reply:
I copied it from your tests 😄